distribution/packages/build.gradle

```diff
@@ -105,6 +106,9 @@ addProcessFilesTask('rpm', false, false)
 // is the same
 Closure commonPackageConfig(String type, boolean oss, boolean jdk) {
   return {
+    onlyIf {
+      OS.current().equals(OS.LINUX)
```

Comment:
What about Mac? Does OSX have the same issue? Perhaps we should just explicitly _exclude_ Windows since as I understand it this would make it impossible to run DEB/RPM packaging tests locally via Vagrant on a Mac.